index.html

```diff
@@ -2785,51 +2784,60 @@ <h3>Consumption</h3>
 fractional component
               </td>
               <td>
-<a>integer</a>
+An <a>integer</a>.
               </td>
             </tr>
             <tr>
               <td>
-<a data-cite="RFC8259#section-6">JSON Number</a> with a fractional component, or
-when entry value is a <a>double</a> regardless of inclusion of fractional
-component
+<a data-cite="RFC8259#section-6">JSON Number</a> with a decimal and fractional
+component, or when entry value is a <a>double</a> regardless of inclusion of
+fractional component
               </td>
               <td>
-<a>double</a>
+A <a>double</a>.
               </td>
             </tr>
             <tr>
               <td>
 <a data-cite="RFC8259#section-3">JSON Boolean</a>
               </td>
               <td>
-<a data-cite="INFRA#boolean">boolean</a>
+A <a data-cite="INFRA#boolean">boolean</a>.
               </td>
             </tr>
             <tr>
               <td>
 <a data-cite="RFC8259#section-3">JSON null literal</a>
               </td>
               <td>
-<a data-cite="INFRA#null">null</a>
+A <a data-cite="INFRA#null">null</a> value.
               </td>
             </tr>
           </tbody>
         </table>
 
-        <p>
-Implementers consuming JSON are advised to ensure that their algorithms are
-aligned with the <a
+        <p class="advisement">
+All implementers creating <a>conforming consumers</a> that produce JSON
+<a>representations</a> are advised to ensure that their algorithms are aligned
+with the  <a
 data-cite="INFRA#convert-a-json-derived-javascript-value-to-an-infra-value">JSON
-consumption rules</a> in the [[INFRA]] specification.
+conversion rules</a> in the [[INFRA]] specification and the <a
+data-cite="RFC7159#section-6">precision advisements regarding Numbers</a> in the
+JSON [[RFC7159]] specification.
         </p>
+
         <p>
-Note that the <code>@context</code> object member, if present, will not have
-additional processing applied to its value, which will be added verbatim to the
-<a href="#data-model">data model</a>.
+If media type information is available to a <a>conforming consumer</a>, the
+media type value is <code>application/did+json</code>, and the data structure 
+being consumed is a <a>DID document</a>, then the root element MUST be a <a
+data-cite="RFC8259#section-4">JSON Object</a>, and all members of this object
+are entries of the <a>DID document</a>. A <a>conforming consumer</a> for a JSON
```

Comment:
```suggestion
If media type information is available to a <a>conforming consumer</a> and the
media type value is <code>application/did+json</code>: the data structure 
being consumed is a <a>DID document</a> and the root element MUST be a <a
data-cite="RFC8259#section-4">JSON Object</a> where all members of the object
are entries of the <a>DID document</a>. A <a>conforming consumer</a> for a JSON
```

Oh, wait, no... @TallTed was incorrect... @rhiaro had the proper fix!